components/member-profile/active-task/index.js

```diff
@@ -1,19 +1,37 @@
 import PropTypes from 'prop-types';
 import classNames from 'components/member-profile/active-task/active-task.module.scss';
 import { timeWas } from 'helper-functions/time-was';
+import { percentageofDaysRemaining } from 'helper-functions/taskProgress';
+import { estimatedDays } from 'helper-functions/estimated-days';
+import { showProgressIndicatorColour } from 'helper-functions/showProgressIndicator';
```

Comment:
rename to `showProgressIndicatorColour` to `progressIndicatorColor`
Also the file name to `progressIndicatorColor`, so that we can be clear what it's doing.

Reply:
Added now